docs/Makefile

```diff
@@ -28,9 +28,9 @@ clean:
 	rm -rf site
 
 .PHONY: publish
-publish: clean
-	$(MAKE) build
-	cp ./swagger-ui/swagger-ui-main-deploy.js ./site/html/rest-api/swagger-ui-main.js
+publish:
+	rm $(swagger_main_script)
```

Comment:
This line looks unnecessary

Reply:
I know this target is mainly gonna be called in CI which is Linux based but I was reading on this and I wasn't convinced that cp would always on all platforms reliably overwrite the target :thinking: I remember @hkang1 having issues with it on Mac.

Reply:
Here's the man page for `cp` on mac: https://ss64.com/osx/cp.html

Of interest is this part:

> For each destination file that already exists, its contents are overwritten if permissions allow